acquia/acquia-cms-starterkit
# ACMS-1426: Update site studio import for starterkit wrt new site studio package manager

Update site studio import for starter kit wrt new site studio package manager.

Fixes: ACMS-1426

Testing steps:
- Install site with this branch with acquia_cms_enterprise_low_code
- Do not provide site studio keys, it should print message to setup site studio letter
- Install site with this branch with acquia_cms_enterprise_low_code
- Provide site studio keys it should import site studio packages
